src/styles/Books.css

```diff
@@ -0,0 +1,183 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Montserrat&family=Roboto&family=Roboto+Slab&display=swap');
+
+:root {
+  --azure: #0290ff;
+  --black-two: #121212;
+}
+
+.loading {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40vh;
+}
+
+@keyframes spinning {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.loading-circle {
+  width: 5rem;
+  height: 5rem;
+  border: 1rem solid #f3f3f3;
+  border-top: 1rem solid #3498db;
+  border-radius: 50%;
+  animation: spinning 1s ease-in infinite;
+}
+
+.bookshelf {
+  width: 100%;
+  padding: 1.5rem 1.5rem 0 1.5rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.book {
+  font-family: 'Roboto', sans-serif;
+  width: 90%;
+  padding: 1.5rem 1rem;
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  margin: 1.25rem auto;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  border-radius: 4px;
+}
```

Comment:
Great job @Chimoneg27 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. kindly create a single class for the repeating rules and re-use this class wherever needed. You may do this for the following rules. `font-family: 'Montserrat', sans-serif;`, `display: flex;`, and `font-family: 'Roboto Slab', serif;`.